ONBOARDING.md

```diff
@@ -0,0 +1,84 @@
+- *Open Communications:* All communications related to code and decision-making should be publicly accessible to ensure asynchronous collaboration, as necessitated by a globally-distributed community.
+- *Community of Peers:* Individuals participate, not organizations. Roles are equal irrespective of title, votes hold equal weight, and contributions are made on a volunteer basis (even if paid to work on code).
+- *Progress over Perfection:* Struggling with a test failure? Can’t quite figure out why that data structure isn’t doing what you expect? Feeling defeated by the complexities of that massively distributed compute engine? The community idea exists to help each other through difficult problems. So fail early, commit often, ask for help, and don’t be afraid.
+
+### Contributor expectations
+
+Contributors are subject to a code of conduct, and it’s essential that all contributions are original work.
+
+- **Read (6 min)** [the Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct)
+- **Read (6 min)** [the OpenSearch CONTRIBUTING guide](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md)
+  - Please make sure you understand the implications of the [Developer Certificate of Origin](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#developer-certificate-of-origin)
+
+### Resources and links
+
+There are more than 90 [GitHub repositories](https://github.com/opensearch-project/OpenSearch) that are part of the OpenSearch project. Explore the list to get a sense of the project breadth and scope.
+
+- The [roadmap](https://github.com/orgs/opensearch-project/projects/1) provides a single place to keep track of upcoming releases and current projects
+
+Looking for answers? While [the forum](https://forum.opensearch.org/) is always a good option, it’s worth checking these first
+
+- [Frequently Asked Questions](https://opensearch.org/faq)
+- [User documentation](https://opensearch.org/docs/latest/)
+- Developer documentation
+  - Start with `DEVELOPER_GUIDE.md` in the repository you’ll be working in (see [OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) or [OpenSearch Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md) guides)
+  - Most repositories have many other documentation files. To find them, search for Markdown files (`*.md` - [example](https://github.com/search?q=repo%3Aopensearch-project%2FOpenSearch-Dashboards+language%3AMarkdown&type=code&l=Markdown)), either on GitHub or in your editor.
+
+## Set up accounts and access
+
+### GitHub account
```

Comment:
Ideally we could just point to a single GitHub doc. But the GitHub getting started doc has a lot of extra content and noise that distract from the task. So I decided to go with a stripped down list here.

Reply:
maybe put a link to that one to say if you need more information, see a complete version there.